src/gui/dialogs/DialogHandler.hpp

```diff
@@ -4,6 +4,12 @@
 #include "fsm_types.hpp"
 #include "DialogFactory.hpp"
 
+enum class redraw_cmd_t : int {
```

Comment:
note:
`enum class redraw_cmd_t : int {`
int is default so consider just `enum class redraw_cmd_t {`